http-gateway/service/gateway.go

```diff
@@ -0,0 +1,45 @@
+package service
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gorilla/mux"
+	"github.com/kimbellG/tournament/http/controller"
+	"github.com/kimbellG/tournament/http/handler"
+
+	"github.com/joho/godotenv"
+	"google.golang.org/grpc"
+)
+
+func initConfig() error {
+	if err := godotenv.Load(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func StartGateway() {
```

Comment:
I would also add a graceful shutdown. For example imagine I want to stop the program with Ctrl+C. When the program stops you should free all resources (close connections to grpc server for example). Even if you don't need to free resources now (connections are closing automatically for example) you still need to provide a way to free up resources whenever program stops )